app/src/main/java/org/apache/taverna/mobile/data/local/PreferencesHelper.java

```diff
@@ -49,11 +52,19 @@
 
     private static final String PREF_KEY_USER_WEBSITE = "pref_user_website";
 
+    private static final String PREF_KEY_PLAYER_CREDENTIAL = "pref_player_credential";
+
+    public static final String PREF_KEY_PLAYER_URL = "pref_player_url";
+
+    public static final String PLAYER_DEFAULT_URL = "http://139.59.28.12:3000/";
```

Comment:
Could we avoid hard-coding this particular server? I have no idea which server that is.. :)